general/device.hpp

```diff
@@ -236,7 +236,7 @@ class Device
    /** @brief Get the current Device MemoryClass. This is the MemoryClass used
        by most MFEM device kernels to access Memory objects. */
    static inline MemoryClass GetMemoryClass() { return Get().mem_class; }
-   
+
    static void SetGPUAwareMPI(const bool force = true)
    { Get().mpi_gpu_aware = force; }
```

Comment:
@camierjs , I didn't know we had GPU Aware MPI! Is this new? Has it shown performance improvements?